### ISSUE TITLE
[3.0.0] Additional `../../` completely broke dashboards maps build on Jenkins

### ISSUE THREAD

**Issue by peterzhuamazon** [opensearch-project/dashboards-maps#83]
Hi,

* https://github.com/opensearch-project/dashboards-maps/pull/12
* https://github.com/opensearch-project/dashboards-maps/pull/13
* https://github.com/opensearch-project/dashboards-maps/pull/63

The above PRs completely break the builds of maps 3.0.0 on Jenkins.

When we build on jenkins we have OSD/plugins folder, and move your custom_import_map folder into it.
With ../../../../ instead of ../../ the call directly goes to 2 layers before OSD root folder.
This only happens on 3.0 branch as of now.

Thanks.

**Comment by peterzhuamazon** [comment]
Temp unblock: https://github.com/opensearch-project/opensearch-build/pull/2844

**Comment by peterzhuamazon** [comment]
@junqiu-lei @heemin32 Thanks.

**Comment by junqiu-lei** [comment]
Offline talked with @peterzhuamazon about it. From https://github.com/opensearch-project/dashboards-maps/pull/12, in custom-import-map plugin, it added two extra `../../` for importing resources from OSD, that's the root cause for the build failed. Currently it only changed at main branch, which only effected for version 3.0.0. 

In order to make sure custom-import-map plugin can be build for 3.0.0 from main branch, we need to recover it without two extra `../../` at main branch.

In order to achieve development easy set up for custom-import-map plugin, we can move custom-import-map plugin folder to repo's root path like maps-dashboard plugin (unreleased in `feature/new-maps` branch) https://github.com/opensearch-project/dashboards-maps/pull/65. But along with plugin place change, we also need some changes on its tests and scripts for its release.

**Comment by junqiu-lei** [comment]
Since in OpenSearch Build repo the 2.4.0 branch still not cut from main, will raise PR to update the change OpenSearch Build after 2.4.0 cut

**Comment by heemin32** [comment]
@junqiu-lei 2.4.0 release is out. Go ahead.

**Comment by VijayanB** [comment]
@junqiu-lei Can this be closed?

**Comment by junqiu-lei** [comment]
Thanks @VijayanB Yes, closing this issue because we finally made this repo to a standalone plugin repo, no path issue should happen again.
